container/gitpod/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM opensuse/leap:15.3
+
+# only dependencies for CircleCI to be able to load/save the package cache
+# and fix permissions for the unprivileged user we use
+RUN zypper -n in tar gzip sudo system-group-wheel && zypper clean
+
+# these are autoinst dependencies
+RUN zypper install -y gcc-c++ cmake ninja pkgconfig\(opencv\) pkg-config perl\(Module::CPANfile\) pkgconfig\(fftw3\) pkgconfig\(libpng\) pkgconfig\(sndfile\) pkgconfig\(theoraenc\) npm
+
+# openQA dependencies
+RUN zypper install -y rubygem\(sass\) python3-base python3-requests python3-future git-core rsync curl postgresql-devel postgresql-server qemu qemu-kvm qemu-tools tar xorg-x11-fonts sudo make
+
+# openQA chromedriver for Selenium tests
+RUN zypper install -y chromedriver
+
+ENV LANG en_US.UTF-8
+
+ENV NORMAL_USER gitpod
+ENV USER gitpod
+
+# RUN echo "$NORMAL_USER ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+# RUN useradd -r -d /home/$NORMAL_USER -m -g users --uid=1000 $NORMAL_USER
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details